src/styles/blocks/page.scss

```diff
@@ -0,0 +1,36 @@
+.page {
+  font-family: Manrope, Arial, serif;
+  color: $main-text-color;
+  font-size: 16px;
+
+  scroll-behavior: smooth;
+
+  // scrollbar-gutter: stable;
+
+  &__body {
+    margin: 0;
+    min-width: 320px;
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transition: all $transform-duration;
```

Comment:
do not apply transitions to `all` as it may impact performance and is a generally bad practice

Reply:
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x